app/public/js/location_search_map.js

```diff
@@ -60,7 +79,7 @@ function initMap() {
 	var options = {
 	  zoom: 19,
 	  center: user_point,
-	  draggable: false,
+	  draggable: true,
```

Comment:
On the location search map, I can still drag the search area off the display without the display being corrected.  I like that the zoom corresponds with the reload.  The drag should probably do the same.

![image](https://user-images.githubusercontent.com/11838191/43409525-2853387c-93f2-11e8-8346-ee8016386f44.png)

![image](https://user-images.githubusercontent.com/11838191/43409600-62b89f48-93f2-11e8-93a9-5f15b327e47f.png)